app/Http/Routes/AdminRoutes.php

```diff
@@ -121,6 +121,21 @@ public function map(Registrar $router)
                 $router->delete('{incident_template}/delete', 'IncidentController@deleteTemplateAction');
             });
 
+            $router->group(['prefix' => 'subscribers', 'middleware' => 'app.subscribers'], function ($router) {
```

Comment:
Do you want to just disable the add button then if it isn't enabled? If someone adds a new subscriber, it will try and fire off an email for verification purposes.

How would you best like to handle that?